src/utils/DatasetsUtils.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+const CONNECTOR_NAME_AZURE_STORAGE = 'Azure Storage Connector';
+const CONNECTOR_NAME_ADT = 'ADT Connector';
+
+export const STORAGE_ROOT_DIR = '%WORKSPACE_FILE%/';
+
+// Build dataset file location in Azure Storage
+function buildStorageFilePath (datasetId, fileName) {
+  return 'datasets/' + datasetId + '/' + fileName;
+}
+
+// Retrieve dataset file location in Azure Storage
+function getStorageFilePathFromDataset (dataset) {
+  const blobPrefix = dataset?.connector?.parametersValues?.AZURE_STORAGE_CONTAINER_BLOB_PREFIX;
+  if (blobPrefix !== undefined) {
+    return blobPrefix.split(STORAGE_ROOT_DIR).pop();
+  }
+}
+
+// Retrieve file name from dataset information
```

Comment:
If we must support custom connector types, I'd rather have them in a distinct file to prevent projects cloning the webapp from getting conflicts when we make changes in the utils files, but we should save it for later indeed :)